data-jdbc/src/main/java/io/micronaut/data/jdbc/operations/JdbcConfiguration.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.data.jdbc.operations;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.annotation.Internal;
+
+/**
+ * Micronaut Data JDBC configurations.
+ *
+ * @author Denis Stepanov
+ * @since 3.3
+ */
+@ConfigurationProperties("micronaut.data.jdbc")
+@Internal
+public final class JdbcConfiguration {
```

Comment:
Don't think a whole new configuration properties is needed for this. It can just go in https://github.com/micronaut-projects/micronaut-data/blob/master/data-jdbc/src/main/java/io/micronaut/data/jdbc/config/DataJdbcConfiguration.java